pir/cpp/BUILD

```diff
@@ -17,23 +17,97 @@
 package(default_visibility = ["//visibility:public"])
 load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
 
+PIR_DEFAULT_INCLUDES = [".", "com_microsoft_seal/include/SEAL-3.5/"]
+PIR_DEFAULT_COPTS =  ["-std=c++17"]
+
+cc_library(
+    name = "pir_context",
```

Comment:
Why have 3 separate libraries? Would it be simpler to have just one library for client and server?